src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs

```diff
@@ -275,27 +275,34 @@ internal static class StreamHelper
 
         #region Static Methods
 
-        internal static void WriteToStream(Stream input, Stream output, PSCmdlet cmdlet, CancellationTokenSource cts)
+        internal static void WriteToStream(Stream input, Stream output, PSCmdlet cmdlet, CancellationToken cancellationToken)
         {
             if (cmdlet == null)
             {
                 throw new ArgumentNullException(nameof(cmdlet));
             }
 
-            Task copyTask = input.CopyToAsync(output, cts.Token);
+            Task copyTask = input.CopyToAsync(output, cancellationToken);
 
             ProgressRecord record = new ProgressRecord(
                 ActivityId,
                 WebCmdletStrings.WriteRequestProgressActivity,
                 StringUtil.Format(WebCmdletStrings.WriteRequestComplete, 0));
```

Comment:
`WriteRequestComplete` is still used here, but I guess it's fine as the status message will be replaced soon, but it feels to me you don't need to call `StringUtil.Format` considering it will be replaced shortly after.

Reply:
Format removed.

Reply:
I know. It does null-or-empty check.